package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hyperproof/hypersync-sdk",
-  "version": "0.8.17",
+  "version": "0.9.0",
```

Comment:
Still not ready to follow SemVer 100%, but bumping to 0.9.0 here to indicate that we are getting ready to take on early adopters.